ciao-controller/internal/datastore/datastore.go

```diff
@@ -2497,15 +2497,13 @@ func (ds *Datastore) UnMapExternalIP(address string) error {
 
 // GenerateCNCIWorkload is used to create a workload definition for the CNCI.
 // This function should be called prior to any workload launch.
-func (ds *Datastore) GenerateCNCIWorkload(vcpus int, memMB int, diskMB int, key string, password string) {
+func (ds *Datastore) GenerateCNCIWorkload(vcpus int, memMB int, diskMB int, key string) {
 	// generate the CNCI workload.
 	config := `---
 #cloud-config
 users:
   - name: cloud-admin
     gecos: CIAO Cloud Admin
-    lock-passwd: false
```

Comment:
I think we should set this explicitly to true otherwise ucd wont lock the password

Reply:
Okay. Fixed!